tests/test_cfp.py

```diff
@@ -189,7 +189,7 @@ def test_if_user_can_add_a_speaker_to_a_proposal(user_client):
             "users_given_name": "Joe",
             "users_family_name": "Doe",
             "phone": "+48523456789",
-            "gender": "M",
+            "gender": "m",
```

Comment:
See above. Not needed.

Reply:
Here's the ticket for getting the users to fill in the new field: https://github.com/EuroPython/epcon/issues/1210

Reply:
Got it.